test/integration/controller/job/suite_test.go

```diff
@@ -91,9 +91,10 @@ func managerAndSchedulerSetup(opts ...job.Option) framework.ManagerSetup {
 		err = job.SetupWebhook(mgr, opts...)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-		sched := scheduler.New(queues, cCache, mgr.GetClient(), mgr.GetEventRecorderFor(constants.AdmissionName))
+		sched := scheduler.New(queues, cCache, mgr.GetClient(), mgr.GetEventRecorderFor(constants.AdmissionName), false)
 		go func() {
```

Comment:
This doesn't need to be a routine anymore, right?

Reply:
yes, has removed routine